packages/ffe-formatters/src/formatPercentage.js

```diff
@@ -10,7 +11,7 @@ export default function formatPercentage(amount, opts = defaults) {
     }
 
     const percentage = amount.toLocaleString('nb-NO', {
-        minimumFractionDigits: 0,
+        minimumFractionDigits: opts.minDecimals,
```

Comment:
It probably does not have any significance but for people already passing an `opts` object with just `maxDecimals`, `minimumFractionDigits` are going to be passed as `undefined` since the default arg won't get used. AFAICT it's treated the same as `0` by `Number.toLocaleString` but in case there are edge-cases maybe we should be doing

```javascript
export default function formatPercentage(amount, opts = {}) {
    if (typeof amount !== 'number') {
        return '';
    }

    const percentage = amount.toLocaleString('nb-NO', {
        minimumFractionDigits: opts.minDecimals || defaults.minDecimals,
        maximumFractionDigits: opts.maxDecimals || defaults.maxDecimals,
    });
    return `${percentage.replace('.', ',')}${NON_BREAKING_SPACE}%`;
}
```

Reply:
I agree on that this way of setting default values for `opts` is flawed and that we should fix it.
But also I approve and merge this pull request as this flaw was there from before.